### ISSUE TITLE
composition of two video one with alpha channel and one without of that.

### ISSUE THREAD

**Issue by commando24** [mdejong/AVAnimator#25]
Can i use this library to compose two video one with alpha channel and one without alpha channel. I know AVFoundation don't support this kind of composition. Please guide me in a direction.

Any help is very appreciated.

**Comment by mdejong** [comment]
Yes, See the Fireworks example or the AVRender example: http://www.modejong.com/AVAnimator/examples.html#Fireworks
